Dockerfile

```diff
@@ -34,24 +37,33 @@ chown -R root:root /opt/pseudos/;                                  \
 chmod -R +r /opt/pseudos/
 
 # Install Python packages needed for AiiDA lab.
-RUN pip install                    \
-    'aiidalab==v20.05.0b1'         \
+RUN pip install 'aiidalab==v20.05.0b1'
+
+# Installing Jupyter-related things in the root environment.
+RUN /usr/bin/pip3 install          \
     'jupyterhub==0.9.4'            \
     'jupyterlab==0.35.4'           \
-    'nbserverproxy==0.8.8'
-
-# Enable nbserverproxy extension.
-RUN jupyter serverextension enable --sys-prefix --py nbserverproxy
+    'fileupload==0.1.5'            \
+    'nbserverproxy==0.8.8'         \
+    'appmode-aiidalab==0.5.0.1'    \
+    'notebook==5.7.8'              \
+    'nglview'                      \
```

Comment:
just out of curiosity: why is this needed here?

Reply:
this was a workaround to be able to run https://github.com/aiidalab/aiidalab-docker-stack/blob/821e78c00677e5f6139375da1c633cc643524657/Dockerfile#L57
which, in turn, was necessary to have `nglview` working.

Reply:
To be clear: I am aware that this is a hack. Not sure, though, what is the better way to achieve the same. One needs to play around with it.

Reply:
done